src/core/StackExchange.Redis.Extensions.Core/Implementations/RedisCacheConnectionPoolManager.cs

```diff
@@ -175,7 +175,7 @@ public void Invalidate()
 
                 this.multiplexer.ConnectionFailed -= this.ConnectionFailed;
                 this.multiplexer.ConnectionRestored -= this.ConnectionRestored;
-                this.multiplexer?.Dispose();
+                this.multiplexer?.Close();
```

Comment:
What's the difference?

Looking here https://github.com/StackExchange/StackExchange.Redis/blob/main/src/StackExchange.Redis/ConnectionMultiplexer.cs#L2693-L2698 `Dispose` is calling `Close`.

Doesn't it?

Reply:
Didnt took a look inside this implementation and missed this moment. 
During testing I made mistake thats why I thought that it also need to change.
I'll update it now